### ISSUE TITLE
Bump dry-types version in rom-core

### ISSUE THREAD

**Issue by DamirSvrtan** [rom-rb/rom#502]
Hi! 👋 

I've already been using dry-validation and dry-types and both are on their newest versions.

I'm looking to introduce rom in the project but dry-validations and rom-core have dependency conflicts because of dry-types.

```
The newest version of dry-validation, 0.12.1, depends on dry-types (~> 0.13.1)
The newest version of rom-core, 4.2.1 depends on dry-types (~> 0.12.1)
```

Is there a possibility to bump the dependency version on rom-core for dry-types to the newest version?

**Comment by DamirSvrtan** [comment]
Thank you for keeping nice Changelogs on both dry-rb and rom-rb repos!

Downgrading dry-validation did the fix, however as much as I understand, upgrading rom gems to the new dry-types API (Integer vs Int) is an ongoing effort. 

Thank you for your work!

**Comment by gogovan-victorlam** [comment]
Any plan to make a new release?
I am using `dry-system` and `dry-validation` and got the same issue
And I found that `master` branch already revised to depends on `dry-types (~> 0.13.0)`

I would like to wait for the new release so that I don't have to downgrade both gems
Thanks!

**Comment by Cervajz** [comment]
Same over here :/

**Comment by solnic** [comment]
I'll have time for rom during this month, and I plan to wrap up 5.0.0 quickly. We've decided to make 5.0.0 a much smaller release than initially planned. I'll publish a post about it on our forum with more info soon.

**Comment by solnic** [comment]
Here's info about updated 5.0.0 roadmap https://discourse.rom-rb.org/t/changed-5-0-0-roadmap/278

**Comment by alexandru-calinoiu** [comment]
Looking forward to the release for this one.

**Comment by solnic** [comment]
Current master works with soon-to-be-released dry-types 1.0.0, so this can be closed. We're wrapping up 5.0.0 release as well.
